Formula/imgur-album-downloader.rb

```diff
@@ -0,0 +1,19 @@
+class ImgurAlbumDownloader < Formula
+  desc "Command Line Utility to Download Imgur Albums"
+  homepage "https://github.com/sgsunder/imgur-album-downloader"
```

Comment:
This gives a 404; should probably be `https://github.com/alexgisby/imgur-album-downloader` instead.